examples/00-load/create-sphere.py

```diff
@@ -0,0 +1,239 @@
+"""
+.. _create_sphere:
+
+Create Sphere Mesh Multiple Ways
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example shows how to create meshes in different ways.
+
+"""
+# sphinx_gallery_thumbnail_number = 5
+
+# %%
+from __future__ import annotations
+
+import numpy as np
+
+import pyvista as pv
```

Comment:
This creates an empty block, not sure if there's a way to format it differently. 
![image](https://github.com/user-attachments/assets/31586a0f-ef8e-48f4-aea2-b15268a4a41e)

Reply:
fixed! 
<img width="823" alt="image" src="https://github.com/user-attachments/assets/83451667-ad1d-4cb6-9c0c-19782d70391a">